kumascript/macros/CSSSyntax.ejs

```diff
@@ -44,6 +44,8 @@ let data = {
 let slug = env.slug;
 let locale = env.locale;
 let name = $0 || (slug ? slug.split("/").pop().toLowerCase() : "preview-wiki-content");
+// deal with conflicting documents for now
```

Comment:
This comment is cryptic. Is it a note-to-self? 
Can you explain *why* certain 32-length md5 looking hashes need to be removed? Perhaps a bit of context and history for a future reader.

Reply:
Hopefully this will be removed soon. But I'll explain it better.